### ISSUE TITLE
`get_channel_metadata()` does not handle lists where single element has error

### ISSUE THREAD

**Issue by mabrownnyu** [SMAPPNYU/youtube-data-api#7]

**Comment by yinleon** [comment]
Does this mean when one element of an entry is invalid, the entire call fails?

**Comment by yinleon** [comment]
Be sure to post a traceback of the error next time!

**Comment by mabrownnyu** [comment]
Yes, the entire call fails, and I forgot to copy the traceback (this was more of a note for myself) so I'll make sure I do that next time

**Comment by yinleon** [comment]
Do you have the input of channel IDs? The get_video_metadata funciton also fails when an ID has an unexpected character (from parsing).

**Comment by yinleon** [comment]
Working on a workaround!
